skyportal/handlers/api/followup_request.py

```diff
@@ -218,7 +222,7 @@ def post(self):
 
         data = self.get_json()
 
-        with DBSession() as session:
+        with self.Session() as session:
             try:
                 assignment_id = post_assignment(
```

Comment:
I'm actually not really sure why we need `post_assignment`. It seems somewhat long but it could live happily inside the POST function (in my opinion). 
In any case, there seems no reason to pass the `user_id`, you can just use the `session.user_or_token`.

Reply:
Also if you just put that code in the handler you can `self.error` what ever you want instead of catching all these types of exceptions.

Reply:
Yeah the issue is the observation_plan handler also calls that function, so we can remove the user stuff but it does need to remain separate.